libs/utils/include/utils/compiler.h

```diff
@@ -158,14 +158,9 @@
 #endif
 
 #if defined(_MSC_VER) && _MSC_VER >= 1900
-#       define UTILS_HAS_FEATURE_CXX_THREAD_LOCAL 1
+#   define UTILS_HAS_FEATURE_CXX_THREAD_LOCAL 1
```

Comment:
Do we still need this define? It doesn't look like it's getting used elsewhere.

Reply:
Yes it's not used right now, but I guess if we need some thread local stuff in the common code, it might still be useful.